lxd/db/errors.go

```diff
@@ -16,4 +16,7 @@ var (
 	// isn't found so we don't abuse sql.ErrNoRows any more than we
 	// already do.
 	ErrNoSuchObject = fmt.Errorf("No such object")
+
+	// ErrNoNode is used to indicate no node has been found for a resource.
+	ErrNoNode = fmt.Errorf("No node found")
```

Comment:
We've tried to stay away from exposing `node` as a term to users, instead using `cluster member`.
It's really just terminology but `cluster member` is far easier to lookup and Google than the much more generic `node`.

Similarly for targeting and location reporting in LXD, we use `--target` and `location` columns/fields instead of using `node` for both of those. The benefit there being that we can re-purpose those to align with AZs or other concepts when scaling LXD past a cluster size.

Reply:
Makes sense